app/posts/design-system-day-2023.md

```diff
@@ -0,0 +1,104 @@
+---
+title: Notes from Design System Day 2023
+description: TODO
+date: 2023-09-23
+author:
+  name: Frankie Roberto
+  url: https://github.com/frankieroberto
+---
+
+This month I was thrilled to take part in [Design System Day 2023](https://design-system.service.gov.uk/community/design-system-day-2023/), a 2-day conference run by the GOV.UK Design System team to collaborate and share knowledge about design systems.
+
+Here are my notes from the conference, as well as an outline of the talk I gave.
+
+## Day 1
+
+The first day was held in-person at Our Dynamic Earth in Edinburgh, and included sessions on research operations, the [Scottish Government Digital Design System](https://designsystem.gov.scot) and trauma-informed research. I also took part in a panel discussion on how design systems manage contributions, where we discussed how and why to encourage teams using design systems to contribute back.
+
+The final session of the day was a keynote by Tash Willcocks called ‘Design systems don’t solve problems, people do’ (love the Goldie Lookin Chain reference!) – where Tash implored design system teams to continually "socialise" their design systems by being in the spaces where teams using the design hang out, and making the effort to understand the reasons why a design might not be being adopted.
```

Comment:
Do you have a link for Task Willcocks’ website?

```suggestion
The final session of the day was a keynote by Tash Willcocks called ‘Design systems don’t solve problems, people do’ (love the Goldie Lookin Chain reference!) – where Tash implored design system teams to continually "socialise" their design systems by being in the spaces where teams using the design hang out, and making the effort to understand the reasons why a design might not be being adopted.
```

Reply:
She doesn’t seem to have one...